packages/cli/commands/project/open.js

```diff
@@ -0,0 +1,70 @@
+const open = require('open');
+const {
+  addAccountOptions,
+  addConfigOptions,
+  getAccountId,
+  addUseEnvironmentOptions,
+  addTestingOptions,
+} = require('../../lib/commonOpts');
+const { loadAndValidateOptions } = require('../../lib/validation');
+const { i18n } = require('@hubspot/cli-lib/lib/lang');
+const { logger } = require('@hubspot/cli-lib/logger');
+const {
+  getProjectConfig,
+  getProjectDetailUrl,
+  verifyProjectExists,
+} = require('../../lib/projects');
+const { projectNamePrompt } = require('../../lib/prompts/projectNamePrompt');
+
+const i18nKey = 'cli.commands.project.subcommands.open';
+
+exports.command = 'open';
+exports.describe = i18n(`${i18nKey}.describe`);
+
+exports.handler = async options => {
+  await loadAndValidateOptions(options);
+
+  const accountId = getAccountId(options);
+  const { project } = options;
+  const { projectConfig } = await getProjectConfig();
+  let projectName = project;
+  if (projectName) {
+    const projectExists = await verifyProjectExists(accountId, projectName);
+    if (!projectExists) {
+      return;
```

Comment:
Instead of just returning, I think it would be better to log out some message along the lines of "The project <project> could not be found in the account <accountName>", and then exiting via `process.exit(EXIT_CODES.ERROR)`

Reply:
We get a log message if its a missing project from the `verifyProjectExists` util (will change to `ensureProjectExists`), didn't want to double up on messaging here. But good call on the process.exit, will add those in